pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1241,6 +1258,21 @@ func Convert_v1_VirtualMachineInstance_To_api_Domain(vmi *v1.VirtualMachineInsta
 		}
 
 	}
+	// Set SEV launch security parameters: https://libvirt.org/formatdomain.html#launch-security
+	if c.UseLaunchSecurity {
+		if !vmi.IsBootloaderEFI() {
```

Comment:
Right, I think I decided to keep it as a precaution measure. Just to have a meaningfull error message in case the admition code is reworked at some point (e.g. if someone accidentally changes the validation logic). Do you think it would be better to drop it from here?

Reply:
I'd say it's fine to leave it, but the error message could be a bit more dramatic! Like `/!\ This should never happen!! /!\ A VMI with SEV + BIOS was admitted!!` :)

Reply:
actually after thinking a bit more about it decided not to be that paranoid and dropped the check :)